geps/gep-1709.md

```diff
@@ -37,8 +45,304 @@ which implementations can prove they satisfy and be recognized for.
+> but when used in conjunction with `Profile` this will result in a report that
+> the profile is not valid for reporting. Implementations in this state may be
+> able to report themselves as "in progress", see the
+> [certification section](#certification) for details.
+
+Alternatively for an `Extended` conformance profile where not all of the
+features are implemented (as described in the [profiles](#profiles) section
+above):
+
+```console
+$ go test ./conformance/... -args \
+    -gateway-class=acme \
+    -conformance-profiles=Layer7Extended,Layer4Core \
+    -unsupported-features=HTTPResponseHeaderModification,HTTPRouteMethodMatching,HTTPRouteQueryParamMatching,
+```
+
+Or the equivalent configuration using the Golang library:
+
+```go
+cSuite, err := suite.New(suite.Options{
+    GatewayClassName: "acme",
+    Profiles: sets.New(
+        Layer7ExtendedProfile,
+        Layer4Core,
+    ),
+    UnsupportedFeatures: sets.New(
+        suite.SupportHTTPResponseHeaderModification,
+        suite.SupportHTTPRouteMethodMatching,
+        suite.SupportHTTPRouteQueryParamMatching,
+    ),
+    // other options
+})
+require.NoError(t, err, "misconfigured conformance test suite")
+cSuite.Setup(t)
+
+for i := 0; i < len(tests.ConformanceTests); i++ {
+    test := tests.ConformanceTests[i]
+    test.Run(t, cSuite)
+}
+```
+
+> **NOTE**: `UnsupportedFeatures` must match features that are extended. This
+> option is invalid when used on a `Core` type profile.
+
+> **NOTE**: In the future we may consider expanding the options to configure
+> and run the conformance test suite using a container image and/or via a
+> configuration file, however it was decided to hold off from doing any of
+> those in this iteration to avoid it getting to big (see the
+> [alternatives](#alternatives-considered) section for more thoughts).
+
+Once an implementation has integrated with the conformance test suite, they can
+move on to [certification](#certification) to report the results.
+
+[go-test]:https://go.dev/doc/tutorial/add-a-test
+[go]:https://go.dev
+[lib]:https://pkg.go.dev/sigs.k8s.io/gateway-api@v0.6.1/conformance/utils/suite
+
+### Certification
+
+Implementations will be able to report their conformance testing results using
+our [reporting process](#reporting-process). Implementations will be able to
+visibly demonstrate their conformance results on their downstream projects and
+repositories using our [certification process](#certification-process).
+
+#### Reporting Process
+
+When conformance tests complete a log message will be emitted:
+
+```console
+Success! Conformance profiles "Layer7Extended" and "Layer4Core" passed for version v0.7.0
+The following extended features were flagged as not supported for "Layer7Extended":
+- HTTPResponseHeaderModification
+- HTTPRouteQueryParamMatching
+```
+
+For the above output upstream Gateway API can manually generate or update a report
+in the following format:
```

Comment:
worth versioning this schema? in case of future changes that might need a migration etc.?

Reply:
Yes, I was just thinking about this today too!

Added in b783fcac4db87482cc0f9f43e19828e237d01de3 :+1: